ci/build_container/build_recipes/luajit.sh

```diff
@@ -36,16 +45,33 @@ index f7f81a4..e80c7ff 100644
  #
  # Disable the JIT compiler, i.e. turn LuaJIT into a pure interpreter.
  #XCFLAGS+= -DLUAJIT_DISABLE_JIT
-@@ -564,7 +564,7 @@ endif
-
+@@ -229,6 +229,7 @@ TARGET_ALDFLAGS= $(LDOPTIONS) $(TARGET_XLDFLAGS) $(TARGET_FLAGS) $(TARGET_LDFLAG
+ TARGET_ASHLDFLAGS= $(LDOPTIONS) $(TARGET_XSHLDFLAGS) $(TARGET_FLAGS) $(TARGET_SHLDFLAGS)
+ TARGET_ALIBS= $(TARGET_XLIBS) $(LIBS) $(TARGET_LIBS)
+ 
++$(warning $(TARGET_CC) $(TARGET_TCFLAGS) -E lj_arch.h -dM)
+ TARGET_TESTARCH=$(shell $(TARGET_CC) $(TARGET_TCFLAGS) -E lj_arch.h -dM)
+ ifneq (,$(findstring LJ_TARGET_X64 ,$(TARGET_TESTARCH)))
+   TARGET_LJARCH= x64
+@@ -251,7 +252,7 @@ ifneq (,$(findstring LJ_TARGET_MIPS ,$(TARGET_TESTARCH)))
+   endif
+   TARGET_LJARCH= mips
+ else
+-  $(error Unsupported target architecture)
++  $(error Unsupported target architecture $(TARGET_TESTARCH))
```

Comment:
It's hard to read diffs, but do we need anything in this new diff block? Maybe I'm missing what we are changing here.

Reply:
I think these crept in because I used a dirty version of the Makefile (does LuaJIT rewrite Makefiles?!). I will back these changes out.

Reply:
No, this was my debug stuff, removed.